docs/index.md

```diff
@@ -3,174 +3,148 @@
-  operator_role_prefix = var.operator_role_prefix
-  account_role_prefix = var.account_role_prefix
-}
-
-module operator_roles {
-  source = "terraform-redhat/rosa-sts/aws"
-  version = "0.0.4"
-
-  create_operator_roles = true
-  create_oidc_provider = true
-  create_account_roles = false
-
-  cluster_id = ocm_cluster_rosa_classic.rosa_sts_cluster.id
-  rh_oidc_provider_thumbprint = ocm_cluster_rosa_classic.rosa_sts_cluster.sts.thumbprint
-  rh_oidc_provider_url = ocm_cluster_rosa_classic.rosa_sts_cluster.sts.oidc_endpoint_url
-  operator_roles_properties = data.ocm_rosa_operator_roles.operator_roles.operator_iam_roles
-}
-
-```
\ No newline at end of file
+    ```
+
+## Terraform examples
+
+The following examples show complete, usable Terraform examples. You must configure the examples with your own credentials. After the files are configured, run the following commands to test these examples:
+
+1. `terraform init` - This command uses all the necessary providers for your project.
+1. `terraform plan -out <name>.tfplan` - **Optional**: This command creates the Terraform plan file that you use to create your infrastructure. This command serves as a "dry run" to ensure that the Terraform will execute properly.
+1. `terraform apply "<name>.tfplan"` - This command executes the Terraform process and creates the needed resources to produce your desired infrastructure.
+
+The example Terraform files are all considered in development:
+
+1. **Required**: [Account Roles terraform](https://github.com/terraform-redhat/terraform-provider-ocm/blob/a42779d6b6712f4dde358344f44b782e4dfcd120/examples/create_rosa_cluster/create_rosa_sts_cluster/classic_sts/account_roles/README.md)
+1. [Create a ROSA cluster that usess STS and has a managed OIDC configuration](https://github.com/terraform-redhat/terraform-provider-ocm/blob/529b10c22c13810b3edbaa01327c7dfdcc207650/examples/create_rosa_cluster/create_rosa_sts_cluster/oidc_configuration/cluster_with_managed_oidc_config/README.md)
+1. [Create a ROSA cluster that uses STS and has an unmanaged OIDC configuration](https://github.com/terraform-redhat/terraform-provider-ocm/blob/529b10c22c13810b3edbaa01327c7dfdcc207650/examples/create_rosa_cluster/create_rosa_sts_cluster/oidc_configuration/cluster_with_unmanaged_oidc_config/README.md)
+
+### Create your operator IAM roles prefix name
+
+The operator IAM roles will be created per cluster with the [rosa-sts](https://registry.terraform.io/modules/terraform-redhat/rosa-sts) module.
+
+## Variables for OCM Terraform Provider
+
+The following variables that you use to create a ROSA cluster with the Terraform provider.
+
+- `client_id` (String) OpenID client identifier.
+- `client_secret` (String, Sensitive) OpenID client secret.
+- `insecure` (Boolean) When set to 'true' enables insecure communication with the server. This disables verification of TLS certificates and host names and it isn't recommended for production environments.
```

Comment:
Second sentence should have either a comma before the "and" or remove the "it" in the second sentence. Also, remove the contraction.